src/test/groovy/com/github/vkorobkov/jfixturescmd/PromptTest.groovy

```diff
@@ -18,7 +18,7 @@ class PromptTest extends Specification {
         when:
         write()
         then:
-        2 * printStream.write(_)
-        2 * printStream.flush()
+        4 * printStream.write(_)
```

Comment:
no we cannot remove, bet we can replace it with 
(1.._) * printStream.write(_)
(1.._) * printStream.flush()
than means: at least 1 time or more